libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -188,6 +236,12 @@ bool ASTJsonConverter::visit(Block const& _node)
 	return true;
 }
 
+bool ASTJsonConverter::visit(PlaceholderStatement const& _node)
+{
+	addJsonNode(_node, "Placeholder", {}, true);
```

Comment:
Placeholder cannot have children. Does the ast contain `children: []` for it? As a refactoring, we could remove this distinction and just clean out all empty children nodes in the json as a post-processing step.